src/components/fb/CreateFidelityBond.jsx

```diff
@@ -509,7 +509,7 @@ const CreateFidelityBond = ({ otherFidelityBondExists, accountBalances, totalBal
 
   return (
     <div className={styles.container}>
-      {alert && <Alert {...alert} className="mt-0" onDismissed={() => setAlert(null)} />}
+      {alert && <Alert {...alert} className="mt-0" onClose={() => setAlert(null)} />}
```

Comment:
I assume there is no big difference between `onDismissed` and `onClose`?

Reply:
This is just trying to be more consistent with the react-bootstrap components.
Argument is now the original arg + a string message: `export type SimpleMessageAlertProps = BsAlertProps & { message: string }`

Otherwise users would be able to pass both `onClose` and `onDismissed` with just the latter being actually used.